### PR TITLE
Mp.fix.test.snapshot.mag 3043

### PR DESCRIPTION
**Pull request by eTheMan80**
# <MAG-3043 / onboarding ui>

## TL;DR

-   Update snapshot tests
-  
-  
-  

## Details

Checklist / Reminders:

-   [x] Documentation / Story is created / adapted
-   [x] Unit tests
-   [ ] CSS changes are BEM-compliant

### CODE REVIEW

File: src/components/Molecules/OnboardingButton/__snapshots__/OnboardingButton.test.tsx.snap [hunk 1]
```diff
@@ -135,7 +135,7 @@ exports[`OnboardingButton should render an anchor 1`] = `
 </button>
 `;
 
-exports[`OnboardingButton should render the onboarding button 1`] = `
+exports[`OnBoardingButton should render the onboarding button 1`] = `
```
Comment (tricaalexandru) [inline src/components/Molecules/OnboardingButton/__snapshots__/OnboardingButton.test.tsx.snap:138]: @eTheMan80 this name is weird :) 
It's not On Boarding Button, it's Onboarding Button. Is there a reason for OnBo... bit?

**Review by tricaalexandru — APPROVED** [review]

File: src/components/Molecules/OnboardingButton/__snapshots__/OnboardingButton.test.tsx.snap (hunk 1 above)
Comment (matthill8286) [reply src/components/Molecules/OnboardingButton/__snapshots__/OnboardingButton.test.tsx.snap:138]: it looks like the changes you made to the name aren't in this version?? Or the snapshots would be correct.

Comment (eTheMan80) [reply src/components/Molecules/OnboardingButton/__snapshots__/OnboardingButton.test.tsx.snap:138]: @tricaalexandru the snapshot should have updated that to `OnboardingButon`

File: src/components/Atoms/Badge/__snapshots__/Badge.test.tsx.snap [hunk 2]
```diff
@@ -17,8 +17,11 @@ exports[`Badge should render 1`] = `
 
 .c3 {
   border: 1px solid #4bcade;
-  background-color: #4bcade;
+  background-color: #ffffff;
```
Comment (matthill8286) [inline src/components/Atoms/Badge/__snapshots__/Badge.test.tsx.snap:20]: @eTheMan80 background-color is in here twice.

File: src/components/Atoms/Badge/__snapshots__/Badge.test.tsx.snap [hunk 3]
```diff
@@ -158,8 +161,11 @@ exports[`Badge should render a badge line 1`] = `
...
+  color: #4bcade;
+  background-color: #494745;
```
Comment (matthill8286) [inline src/components/Atoms/Badge/__snapshots__/Badge.test.tsx.snap:166]: @eTheMan80 And here as well.

**Review by matthill8286 — CHANGES_REQUESTED** [review]

File: src/components/Atoms/Badge/__snapshots__/Badge.test.tsx.snap (hunk 3 above)
Comment (eTheMan80) [reply src/components/Atoms/Badge/__snapshots__/Badge.test.tsx.snap:166]: @matthill8286 I didn't update that component, I just updated the snapshots.

File: src/components/Atoms/Badge/__snapshots__/Badge.test.tsx.snap (hunk 2 above)
Comment (eTheMan80) [reply src/components/Atoms/Badge/__snapshots__/Badge.test.tsx.snap:20]: @matthill8286 I didn't update that component, I just updated the snapshots.

**Review by matthill8286 — APPROVED** [review]

**Pull request merged**